src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrCodeflower.cs

```diff
@@ -400,8 +400,16 @@ protected async Task<bool> UpdateDependenciesAndToolset(
                 _fileSystem.DeleteDirectory(commonDir, true);
             }
 
+            // Check if the VMR contains src/arcade/eng/common
+            var arcadeEngCommonDir = sourceRepo / VmrInfo.SourceDirName / "arcade" / Constants.CommonScriptFilesPath;
```

Comment:
I think this is wrong. Forward flow and backflow need to behave differently. Backflow will use this path while forward flow will use usual one.